components/ShoppingNav.js

```diff
@@ -13,8 +14,10 @@ export default function Nav() {
     <>
       {hydrated && (
         <NavWrapper>
-          <Link passHref href="/shopping">
-            <SmallButton />
+          <Link passHref href="/">
+            <SmallButton>
+              <IconLeft />
```

Comment:
User experience-wise it's a little unexpected to get 'logged out' by clicking this button. I'd probably replace this with an actual logout button, if that's the intended use?

Reply:
good idea!